plugins/inputs/snmp/snmp.go

```diff
@@ -346,7 +375,6 @@ func (s *Snmp) Gather(acc telegraf.Accumulator) error {
 	if err := s.init(); err != nil {
 		return err
 	}
-
```

Comment:
It is better not to remove this blank line.

Reply:
sure